admin/sales_analytics/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.conf.urls import url
+
+from . import views
+
+urlpatterns = [
+    url(r'^dashboard$', views.dashboard, name='dashboard'),
+    url(r'^user_session', views.user_session, name='user_session'),
+    url(r'^products_view', views.products_view, name='products_view'),
+    url(r'^products_usage', views.products_usage, name='products_usage'),
+    url(r'^debug_test', views.debug_test, name='debug_test'),
```

Comment:
Currently all keen related analytics such as `user_session`, `products_view` and `debug_test` are turned off for staging by setting `KEEN_READY` to `False`, but the code remains there. I had planned the last one to serve for user customized queries. Maybe rename it? Or should we remove all keen stuff and only leave dashboard there?